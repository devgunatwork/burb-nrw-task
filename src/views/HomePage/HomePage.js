import React from 'react';
import Typography from '@material-ui/core/Typography';
import StyledLink from '../../components/StyledLink';
import styles from './HomePage.module.scss';

/**
 * @description component is a landing page
 * */
const HomePage = () => (
  <div className={styles.root}>
    <Typography variant="h1">Shows Available</Typography>
    <Typography variant="h4">
      <StyledLink to="/shows/6771">The Powerpuff girls</StyledLink> <br />
      <StyledLink to="/shows/38963">The Mandalorian</StyledLink>
    </Typography>
  </div>
);

export default HomePage;
