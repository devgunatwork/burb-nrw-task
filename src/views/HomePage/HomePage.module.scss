@import "src/style/mixins/mixins";

.root {
  @include pageCenter;
  min-width: 800px;
}
