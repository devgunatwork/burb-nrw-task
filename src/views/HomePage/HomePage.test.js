import React from 'react';
import HomePage from './HomePage';

describe('views', () => {
  describe('HomePage', () => {
    describe('UI', () => {
      it('should match the snapshot', () => {
        const wrapper = shallow(<HomePage />);

        expect(wrapper).toMatchSnapshot();
      });
    });
  });
});
