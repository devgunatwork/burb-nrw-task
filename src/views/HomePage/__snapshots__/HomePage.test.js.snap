// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views HomePage UI should match the snapshot 1`] = `
<div
  className="root"
>
  <WithStyles(ForwardRef(Typography))
    variant="h1"
  >
    Shows Available
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))
    variant="h4"
  >
    <StyledLink
      to="/shows/6771"
    >
      The Powerpuff girls
    </StyledLink>
  </WithStyles(ForwardRef(Typography))>
</div>
`;
